devrequirements.txt

```diff
@@ -7,3 +7,4 @@ sphinx
 # Dependencies
 feedparser
 requests
+lxml
```

Comment:
Is lxml not needed as a normal dependency? This file is only used for development, so `lxml` should also be added to [setup.py](https://github.com/miaowware/ctyparser/blob/master/setup.py) (setup.py being what defines dependencies to install when someone install ctyparser)